src/ensembl/src/content/app/browser/browser-bar/BrowserBar.scss

```diff
@@ -88,3 +88,65 @@
     }
   }
 }
+
+.browserInfoRegion {
+  form {
+    display: inline-block;
+  }
+
+  label {
+    margin-right: 5px;
+  }
+
+  input {
+    padding: 1px 6px;
+    width: 200px;
+  }
```

Comment:
I don't see either `input` or `form` element in the .browserInfoRegion element in the corresponding tsx